### DISCUSSION TITLE
When will Copilot chat be coming to PhpStorm and other JetBrains IDEs?

### DISCUSSION THREAD

**Discussion by MichaelNZ85** [community/community#108242] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I'm on a business subscription, and do not have access to Github Copilot in PhpStorm, my main IDE. All I see is a link to join the waitlist, which informs me I'm not eligible as I'm not on an Individual subscription plan.

When will Copilot chat be available in PhpStorm (and the other JetBrains IDEs like IntelliJ, PyCharm etc)? It's annoying to have to move over to VSCode to use Copilot chat.

**Comment by EcentriaGit** [comment]
I have the same question. The majority of our developers are using PHPStorm. We really want to have the chat feature; it is a significant part of the value we hope to gain from paying for Copilot.

**Reply by MichaelNZ85** [reply]
You can download VSCode and run the chat in that, but it's just annoying having to switch programs rather than highlighting the code in PhpStorm and asking Copilot to help with it.

**Comment by MichaelNZ85** [comment]
Bump.

Any update?

**Comment by MatsSvenssonNewseed** [comment]
Here is my opinion on the subject:
_(Unfortunately you will have to join my private beta to find out you can't read it here anyway.)_

**Comment by MichaelNZ85** [comment]
Bump. Any update?

**Comment by davecheney** [comment, marked as answer]
@MichaelNZ85 currently chat in jetbrains ide's is a waitlisted feature. It is a very popular item and we are adding invites as capacity permits. You can sign up for the waitlist here https://github.com/github-copilot/chat_jetbrains_waitlist_signup/join
